src/Microsoft.AspNetCore.Server.Kestrel/WebHostBuilderKestrelExtensions.cs

```diff
@@ -28,6 +29,7 @@ public static IWebHostBuilder UseKestrel(this IWebHostBuilder hostBuilder)
             return hostBuilder.ConfigureServices(services =>
             {
                 services.AddTransient<IConfigureOptions<KestrelServerOptions>, KestrelServerOptionsSetup>();
+                services.AddTransient<IConfigureOptions<KestrelServerOptions>, KestrelServerConfigureOptions>();
```

Comment:
It's getting fixed: https://github.com/aspnet/DependencyInjection/issues/530
That said, @halter73 says we can't rely on that because it doesn't guarantee the same ordering and KestrelServerConfigureOptions requires KestrelServerOptionsSetup to have run first.